code/core/src/manager/components/sidebar/TagsFilter.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { findByRole, fn } from '@storybook/test';
+
+import { TagsFilter } from './TagsFilter';
+
+const meta = {
+  component: TagsFilter,
```

Comment:
I guess you have to put the title explicitly to get these components to land in the same sidebar entry as the other Sidebar components?

![image](https://github.com/user-attachments/assets/39046fb8-22e5-49b6-9649-c6190e14a8b5) vs ![image](https://github.com/user-attachments/assets/e17fde3d-2a4c-47cf-bb9c-8d850ac56e51)

Reply:
I want our Storybooks to use auto-title as much as possible. I'll move the other stories over in a separate PR.